.gitignore

```diff
@@ -402,6 +402,7 @@ python-package/lightgbm/VERSION.txt
 
 # R build artefacts
 **/autom4te.cache/
+conftest*
```

Comment:
/gha run r-valgrind

Reply:
valgrind is happy!

https://github.com/microsoft/LightGBM/actions/runs/338827520